addons/format_lame.c

```diff
@@ -0,0 +1,553 @@
+/*
+ * Asterisk -- An open source telephony toolkit.
+ *
+ * Copyright (C) 1999 - 2018, Digium, Inc.
+ *
+ * Mark Spencer <markster@digium.com>
```

Comment:
Sorry I was not clear, This should be _your_ copyright, not Digium's or Mark Spencer's. You can include multiple copyrights if you like.

Reply:
Example of multiple copyrights:

https://github.com/asterisk/asterisk/blob/79210720bc928feb26a340752492b3036f5d2e2b/apps/app_originate.c#L4-L8

Reply:
Hope it's fine now.